src/Widgets/AddressWidget/AddressWidgetClass.js

```diff
@@ -2,16 +2,34 @@ import { registerTextExtract } from 'utils/textExtractRegistry';
 
 const AddressWidget = Scrivito.provideWidgetClass('AddressWidget', {
   attributes: {
-    address: 'html',
-    listItems: ['widgetlist', { only: 'AddressListItemWidget' }],
     showLogo: ['enum', { values: ['yes', 'no'] }],
     showBorderBottom: ['enum', { values: ['yes', 'no'] }],
+
+    locationName: 'string',
+    locationStreetAddress: 'string',
+    locationLocality: 'string',
+    locationPostalCode: 'string',
+    locationRegion: 'string',
+    locationCountry: 'string',
+
+    phone: 'string',
+    mobile: 'string',
+    fax: 'string',
+    eMail: 'string',
```

Comment:
Sorry, but I just noticed, that `E-Mail` may be correct in german, but not in english. So it's either `E-mail` or `Email`. I asked already Andreas and Antony and both do recommend `Email`. Could you go over the code-base and change all usages of `E-Mail`, `e-mail` etc. to `Email`/`email`?

Also this attribute just just be `email`.